ethereum/ethstats/src/main/java/org/hyperledger/besu/ethstats/util/NetstatsUrl.java

```diff
@@ -38,7 +40,10 @@ public interface NetstatsUrl {
 
   String getContact();
 
-  static NetstatsUrl fromParams(final String url, final String contact) {
+  @Nullable
+  Path getPemTrust();
```

Comment:
Perhaps this interface is misnamed but including the pemTrust as part of this type representing an URL doesn't make sense to me.

Reply:
I've renamed this interface to `EthStatsConnectOptions` to better represent its usage and intent.